tools/internal_ci/linux/aws/grpc_aws_experiment.cfg

```diff
@@ -15,7 +15,7 @@
 # Config file for the internal CI (in protobuf text format)
 
 # Location of the continuous shell script in repository.
-build_file: "grpc/tools/internal_ci/linux/aws/grpc_aws_experiment.sh"
+build_file: "grpc/tools/internal_ci/linux/aws/grpc_aws_run_remote_test.sh"
```

Comment:
nit: why change the name? It seemed conventional to name these like the cfg.

Reply:
so we now how multiple jobs (and multiple cfg files) and they are all using the same script grpc_aws_run_remote_test.sh (along with setting an env var that determines the remote testing script to run). It wouldn't make sense for all the other language specific cfg files to invoke "grpc_aws_experiment.sh".